Makefile

```diff
@@ -62,7 +62,7 @@ sd-app: prep-salt ## Provisions SD APP VM
 
 sd-whonix: prep-salt ## Provisions SD Whonix VM
 	sudo qubesctl --show-output state.sls sd-whonix
-	sudo qubesctl --show-output --skip-dom0 --targets sd-whonix-buster-template,sd-whonix state.highstate
+	sudo qubesctl --show-output --skip-dom0 --targets whonix-gw-15,sd-whonix state.highstate
```

Comment:
Why was this changed? sd-whonix-buster-template is cloned off of whonix-gw-15

Reply:
I don't have a `sd-whonix-buster-template`. What am i missing?

Reply:
Ahh, right, logging in `sd-whonix`. Good catch, `sd-whonix-buster-template` doesn't exist so it's effectively a no-op here. What will happen to `sys-whonix` ? The package will be installed but since the configuration is not in place, nothing will happen there?